sechub-cli/script/go-gen-supported-source-extensions.sh

```diff
@@ -42,5 +42,5 @@ while read line ; do
   RESPONSE="$RESPONSE/* $SRCLANG */"$'\n'
 done <<< "$SRC"
 
-echo -n "$RESPONSE" | head -n -1  # all but last line
+echo -n "$RESPONSE" | awk 'NR > 1{print p}{p=$0}'  # all but last line
```

Comment:
With this, I relied on [this post](https://stackoverflow.com/questions/18127726/getting-head-to-display-all-but-the-last-line-of-a-file-command-substitution-an). Now that I look at it, it seems like I should be able to replace `awk 'NR > 1{print p}{p=$0}'` with `sed \$d`